src/Parser/Rogue/Subtlety/Modules/Legendaries/FirstOfTheDead.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink'; 
+import ItemLink from 'common/ItemLink'; 
+import Analyzer from 'Parser/Core/Analyzer';
+import ResourceTypes from 'common/RESOURCE_TYPES';
+import Wrapper from 'common/Wrapper';
+import DamageTracker from 'Parser/Core/Modules/AbilityTracker';
+
+class FirstOfTheDead extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    damageTracker: DamageTracker,
+  };
+  
+  totalValue = 0;
+  valueCasts = 0;
+
+	on_initialized(){
+		this.active = this.combatants.selected.hasHands(ITEMS.THE_FIRST_OF_THE_DEAD.id);
+  }
+  
+  on_toPlayer_energize(event) {
+    if(!event.ability) return;
+    if(event.resourceChangeType !== ResourceTypes.COMBO_POINTS) return;
+    const buffID = SPELLS.THE_FIRST_OF_THE_DEAD_BUFF.id;
+    if(!this.combatants.selected.hasBuff(buffID)) return;
+
+    let baseCp = 0;
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.BACKSTAB.id || spellId === SPELLS.GLOOMBLADE_TALENT.id) {
+      baseCp += 1;
+      
+    }
+    else if(spellId === SPELLS.SHADOWSTRIKE.id) {
+      baseCp += 2;
+    }
+    else {
+      //Not affected by this legendary
+      return;
+    }
+
+    if(!this.combatants.selected.hasBuff(SPELLS.SHADOW_BLADES.id)) baseCp += 1;
+
```

Comment:
Shouldn't this be (without the ``!`` to make sure that only counts when they **have** the buff and not when they don't) 
```jsx 
if(this.combatants.selected.hasBuff(SPELLS.SHADOW_BLADES.id)) {
baseCp += 1;
}